modules/income_and_assets/lib/income_and_assets/pdf_fill/va21p0969.rb

```diff
@@ -35,19 +35,56 @@ class Va21p0969 < ::PdfFill::Forms::FormBase
       # The path to the PDF template for the form
       TEMPLATE = "#{IncomeAndAssets::MODULE_PATH}/lib/income_and_assets/pdf_fill/pdfs/#{FORM_ID}.pdf".freeze
 
-      # Each section of the form
-      SECTIONS = [Section1, Section2, Section3, Section4,
-                  Section5, Section6, Section7, Section8,
-                  Section9, Section10, Section11, Section12,
-                  Section13].freeze
+      # Starting page number for overflow pages
+      START_PAGE = 11
+
+      # Map question numbers to descriptive titles for overflow attachments
+      QUESTION_KEY = {
+        1 => 'Veteran name',
+        2 => 'Social Security Number',
+        3 => 'Recurring Income Not Associated with Accounts or Assets',
+        4 => 'Associated Incomes',
+        5 => 'Income and Net Worth Associated with Owned Assets',
+        6 => 'Income and Net Worth Associated with Royalties and Other Properties',
+        7 => 'Asset Transfers',
+        8 => 'Trusts',
+        9 => 'Annuities',
+        10 => 'Assets Previously Not Reported',
+        11 => 'Discontinued or Irregular Income',
+        12 => 'Waiver of Receipt of Income'
+      }.freeze
+
+      # V2-style sections grouping question numbers for overflow pages
+      SECTIONS = [
+        { label: 'Section III: Recurring Income Not Associated with Accounts or Assets', question_nums: [3] },
```

Comment:
I noticed that `SECTIONS` starts at Section III, omitting the first two. Adding those sections would be good for consistency, and it ensures that questions show up in the right place and order.

Side note: I did notice that the 0969 has an unusual 1:1 mapping of questions to sections. Questions 3-12 are each essentially a list-and-loop. But it feels like Questions 1 and 2 should both be broken up into multi-question sections -- e.g. 1A and 1B feel like separate questions with 1A being a multi-field question. We can circle back to this with designers later.